esphome/ds18b20_beta.yaml

```
@@ -0,0 +1,36 @@
+---
+# vindriktning 
```

yamllint

[error] trailing spaces

(trailing-spaces)